plugins/modules/aws_secret.py

```diff
@@ -139,6 +153,29 @@
       returned: always
       type: dict
       sample: { "dc1ed59b-6d8e-4450-8b41-536dfe4600a9": [ "AWSCURRENT" ] }
+    tags:
```

Comment:
Don't we also need to add `version_added` also for this returned parameter (even it's not new)?